netlogo-gui/src/main/app/codetab/CodeTab.scala

```diff
@@ -50,7 +54,20 @@ class CodeTab(val workspace: AbstractWorkspace) extends JPanel
   override def zoomTarget = text
 
   val errorLabel = new EditorAreaErrorLabel(text)
-  val toolBar = getToolBar
+  val toolBar = new ToolBar {
```

Comment:
But we never override it (if, in some point in the future we will, we can change it then).
Regarding the clutter, I don't see

```
val toolbar = getToolbar
def getToolbar = {
  ...
}
```

as more cluttered than

```
val toolbar = {
  ...
}
```

Maybe even a bit less cluttered